src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public boolean getStatus() {
+        return isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getTaskType() {
+        return null;
+    }
+
+    public void printAddDetails(int taskCounter) {
+        System.out.println("The following task has been added:\n[" + getTaskType() +"][" + getStatusIcon() + "] " + getDescription());
+        System.out.println("\nYou've got " + taskCounter + " task(s) in the list!\n");
+    }
+
+    public void printListDetails(int taskCounter) {
```

Comment:
Similarly, consider refactoring the `printListDetails()` function so that you don't need to duplicate part of the print statement in the child classes

Reply:
Good suggestion, I have decided to make task become an abstract class and this method abstract as well, to remove the duplication